jebediah:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position controllers ---------------------------------------
  coxa_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_coxa_1
    pid: {p: 10.0, i: 0.05, d: 0.20}
  femur_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_femur_1_1
    pid: {p: 10.0, i: 0.05, d: 0.20}
  tibia_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_femur_1_2
    pid: {p: 10.0, i: 0.05, d: 0.20}
  coxa_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_coxa_2
    pid: {p: 10.0, i: 0.05, d: 0.20}
  femur_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_femur_2_1
    pid: {p: 10.0, i: 0.05, d: 0.20}
  tibia_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_femur_2_2
    pid: {p: 10.0, i: 0.05, d: 0.20}
  coxa_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_coxa_3
    pid: {p: 10.0, i: 0.05, d: 0.20}
  femur_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_femur_3_1
    pid: {p: 10.0, i: 0.05, d: 0.20}
  tibia_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_femur_3_2
    pid: {p: 10.0, i: 0.05, d: 0.20}
  coxa_4_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_coxa_4
    pid: {p: 10.0, i: 0.05, d: 0.20}
  femur_4_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_femur_4_1
    pid: {p: 10.0, i: 0.05, d: 0.20}
  tibia_4_position_controller:
    type: effort_controllers/JointPositionController
    joint: servo_femur_4_2
    pid: {p: 10.0, i: 0.05, d: 0.20}